index.js

```diff
@@ -17,7 +18,7 @@ Object.defineProperty(exports, 'mapFileCommentRegex', {
 
 
 function decodeBase64(base64) {
-  return new Buffer(base64, 'base64').toString();
+  return buffer.Buffer.from(base64, 'base64').toString();
```

Comment:
please just use `buffer.from` to get the safe-buffer implementation

Reply:
https://github.com/feross/safe-buffer/blob/master/index.js#L16
```javascript
  exports.Buffer = SafeBuffer
```

It *is* the safe-buffer implementation or the travis-ci build on `v0.10` would break.

Reply:
Oops, missed that branch on my review

Reply:
The example `require` that's listed on the `safe-buffer` doesn't help; it does variable aliasing on the name `Buffer`:
```javascript
var Buffer = require('safe-buffer').Buffer
```

Reply:
Ah that's confusing @empjustine. Shadowing a global isn't a good idea.
As you can see @phated and I were confused right off the bat.

Please rename the reference to `SafeBuffer` and I'll go ahead and merge it.

Thanks.